metrics/grafana/tikv_details.dashboard.py

```diff
@@ -8483,6 +8483,18 @@ def BackupLog() -> RowPanel:
                     )
                 ],
             ),
+            graph_panel(
+                title="Initial Scanning Task Status",
+                description="The task status of initial scanning.",
+                targets=[
+                    target(
+                        expr=expr_sum_rate(
```

Comment:
It is a gauge, we don't need to `rate` over it.

Reply:
fixed